### ISSUE TITLE
Duplicate "Payment method successfully added" notices if Stripe plugin active

### ISSUE THREAD

**Issue by dhjw** [Automattic/woocommerce-payments#3707]
### Describe the bug
On the My Account -> Payment Methods page after successfully adding a payment method the success notification is duplicated if the WooCommerce Stripe Gateway plugin is active alongside WooCommerce Payments, so 2 success notices are displayed one on top of the other. (The Stripe Gateway payment method does not have to actually be enabled in WooCommerce Settings)

### To Reproduce
Add a saved credit card with WooCommerce Payments and WooCommerce Stripe Gateway plugins active.

### Expected behavior

One notice.

### Suggested fix

Check if the notice already exists before adding another one or automatically disable the WooCommerce Stripe Gateway plugin when WooCommerce Payments is enabled.

**Comment by nagpai** [comment]
Thanks for reporting. 

I was unable to replicate this issue. 

**Test setup** 
WooCommerce Version 6.7.0
WC Pay - 4.5.0
WC Stripe Gateway  - 6.5.1

**Steps for test done**
1. Install and enable [WC Stripe Gateway](https://wordpress.org/plugins/woocommerce-gateway-stripe/)
2. On an incognito window, browse to `My account page > Payment Methods`
3. Click add payment methods and try to add and save any of the test cards. 

What is seen: 

![image.png](https://images.zenhubusercontent.com/62d6d37561ee1466e2e6958d/0b95dba1-e700-4ba1-a9d3-8c6e30728095)
-----

**Comment by nagpai** [comment]
@dhjw , Can you confirm if you are able to reproduce this issue with latest updated WooCommerce core and extensions? 

It will be also helpful if you  can share the [system status report ( SSR )](https://woocommerce.com/document/understanding-the-woocommerce-system-status-report/) of your test setup.

**Comment by dhjw** [comment]
Note you should install and activate both WooCommerce Payments and the WooCommerce Stripe Gateway plugin to try to reproduce the issue. (The Stripe Gateway payment method does not have to actually be enabled in WooCommerce Settings)

**Comment by nagpai** [comment]
@dhjw I tested with both WC Pay and WC Stripe Gateway Plugin enabled, indeed. I have tried this on Chrome, Safari and Firefox and still unable to reproduce it. 

I also created a separate `customer` account and tried adding a payment method using that and was unable to get the duplicate message there either. 

Have you tested this with all plugins deactivated except, WooCommerce core, WooCommerce Payments, and WooCommerce Stripe Gateway?  I would also advice disabling any browser extensions to rule out any local browser issues.
